src/prepack-cli.js

```diff
@@ -267,15 +267,29 @@ function run(
   }
 
   let errors: Map<BabelNodeSourceLocation, CompilerDiagnostic> = new Map();
+  let errorList: Array<CompilerDiagnostic>;
```

Comment:
Initialize this to the empty list.

Reply:
I am impressed that we have a test for this :-)